tasktiger/__init__.py

```diff
@@ -116,56 +126,45 @@ def init(self, connection=None, config=None, setup_structlog=False):
         self.config = {
             # String that is used to prefix all Redis keys
             'REDIS_PREFIX': 't',
-
```

Comment:
No strong opinion here but it was more readable with the newlines.

Reply:
I'm inclined to agree, but I think the only way to keep them is by creating an unformatted block. I'd rather not create unformatted blocks unless there's a breaking change though.

Reply:
Looks like Black would allow a plain `#` to add a blank line between each setting if we think that helps readability. Your call if that helps or hurts.